main.go

```diff
@@ -133,3 +134,30 @@ func getWalletInfo() {
 	}
 
 }
+
+func getChainTips() {
+	basicAuth := base64.StdEncoding.EncodeToString([]byte(*rpcUser + ":" + *rpcPassword))
+	rpcClient := jsonrpc.NewClientWithOpts("http://"+*rpcHost+":"+*rpcPort,
+		&jsonrpc.RPCClientOpts{
+			CustomHeaders: map[string]string{
+				"Authorization": "Basic " + basicAuth,
+			}})
+	var chaintips *GetChainTips
+
+	for {
```

Comment:
I learned a bit of go while reading this just now. Seeing the `for` keyword made me expect the type of 'for' loop most languages have -- `for` followed by either a triplet of init/condition/increment statements, or an 'each (var) in (collection)' construct. But on looking it up, I see that without those things it's a 'forever' loop.
https://tour.golang.org/flowcontrol/4